static/step0.html

```diff
@@ -3,32 +3,18 @@
   <head>
     <!-- IMPORTANT: Set the character set to UTF-8. Otherwise you may get weird symbols on the score. -->
     <meta charset="utf-8" />
-    <!-- Load Bravura and Academico (see vexflow-fonts for available fonts) -->
-    <style>
-      @font-face {
-        font-family: 'Bravura';
-        src: url(https://cdn.jsdelivr.net/npm/@vexflow-fonts/bravura/bravura.woff2) format(woff2);
-      }
-      @font-face {
-        font-family: 'Academico';
-        src: url(https://cdn.jsdelivr.net/npm/@vexflow-fonts/academico/academico.woff2) format(woff2);
-      }
-    </style>
   </head>
   <body>
-    <span style="font-family: Academico"></span>
-    <span style="font-family: Bravura"></span>     
-
     <!-- Div where the scores will be output -->
     <div id="output"></div>
 
     <!-- Load library -->
-    <script src="https://cdn.jsdelivr.net/npm/vxflw-early-access@5.0.0-alpha.6/build/cjs/vexflow.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vxflw-early-access@alpha/build/cjs/vexflow.js"></script>
 
     <script>
-    document.fonts.ready.then(() => {
-      /* global Vex */
-      Vex.Flow.setMusicFont('Bravura', 'Academico');
+    /* global Vex */
+    Vex.Flow.loadFonts('Bravura', 'Academico').then(() => {
+      Vex.Flow.setFonts('Bravura', 'Academico');
```

Comment:
@ronyeh would it not be better that loadFonts also calls setFonts internally?
This just examples and I will not wait for reviews so that you can focus on vexflow, however I will comment if I see things like that.

Reply:
I prefer they remain separate.

You can call loadFonts on a very long list of fonts.... but then you might just want to call setFonts on just Bravura, Academico.

Then later when your user selects a different font, you can call setFonts on Petaluma, Petaluma Script.

Reply:
ok understood.

Reply:
Also loadFonts() is optional. You can preload the fonts via @font-face and empty span tags, and never have to call loadFonts(). But you should aost always call setFonts().

The only time you don't need to is if you are using "Bravura,Academico" because that is our default.